src/components/datepicker/FullNavigationBar.js

```diff
@@ -0,0 +1,90 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '../box';
+import { Button, ButtonGroup } from '../button';
+import {
+  IconArrowLeftMediumOutline,
+  IconArrowRightMediumOutline,
+  IconArrowLeftSmallOutline,
+  IconArrowRightSmallOutline,
+} from '@teamleader/ui-icons';
+import { Section, Heading2 } from '../..';
+
+const PADDINGS = {
+  small: 3,
+  medium: 4,
+  large: 5,
+};
+
+class FullNavigationBar extends PureComponent {
+  handlePreviousClick = () => {
+    this.props.onPreviousClick();
+    this.props.onClickPrevious();
+  };
+
+  handleNextClick = () => {
+    this.props.onNextClick();
+    this.props.onClickNext();
```

Comment:
Are these double functions just to provide both ways of writing it? 🤔

Reply:
The `onNextClick` and `onPreviousClick` are functions of `react-day-picker` itself that update our calendar. The `onClickNext` and `onClickPrevious` are functions that we will use to update our `displayedMonth`, these functions are necessary to make the custom "today" button work.

Reply:
Can we maybe add a comment here to make this clear? Or destructure from the props and rename it to `onDayPickerNextClick`, otherwise it's a bit confusing

Reply:
Good remark!